lib/js-preprocessor/broccoli-generate-components.js

```diff
@@ -60,8 +60,9 @@ GenerateComponents.prototype.appComponentFor = function(fileName) {
   return "export { default } from '" + defaultModule + "';\n";
 };
 
-var componentsToGenerate = function(baseDir, scss) {
+var componentsToGenerate = function(baseDir, scss, modulePrefix) {
   var scssFiles = walkSync(baseDir, { globs: scss });
+  var baseKindFile = path.join(modulePrefix, 'components', 'ui-kind').replace('modules/', '');
```

Comment:
ah my mistake, you can actually pass in `this.moduleName` and use that as the prefix instead. I passed in both to the broccoli plugin for this reason. This is so we can config this from the index.js file (broccoli shouldn't have knowledge of what ember's namespacing is)